reference/toolstacks/php/drupal8/README.md

```diff
@@ -0,0 +1,93 @@
+# Getting Started
+
+## Drupal 8 and Composer
+
+The recommended way to deploy Drupal 8 on Platform.sh is to use Composer. [Composer](https://getcomposer.org/) is the PHP package management suite, and is now supported by Drupal 8 (and Drupal 7 in a pinch). There is an unofficial but well-supported Composer flavor of Drupal 8 called [Drupal Composer](https://github.com/drupal-composer/drupal-project) that we recommend.  If you use the [Drupal 8 Example Repository](https://github.com/platformsh/platformsh-example-drupal8/) or select the Drupal 8 option when creating a new project from a template, that's what you will be using.  You can also create your own project directly from that repository and add the Platform.sh-specific configuration files.  Note that you will also need to add the Drupal.org Composer repositories to `composer.json` if you are not working from our template.
```

Comment:
The last two sentences seem a bit too much information perhaps? (these are quite long paragraphs, and the note about Drupal.org repositories is perhaps a rarely needed detail)